### PR TITLE
Uber merge back to origin

### PR DESCRIPTION
**Pull request by aknot242**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,35 +16,49 @@ A comprehensive demonstration Ansible-based solution to quickly build and deploy
...
+```
 
+- ssh into your jumphost with the following command
+```bash
+ssh -i <path to privatekeyfile> ubuntu@<ubuntuserver>
+```
+
+- prepare the jumphost to run the ansible playbook with the follow commands
+```bash
+git clone https://github.com/mjmenger/ansible-uber-demo.git
```
Comment (mjmenger) [inline README.md]: This needs to be updated to the new location

File: SUPPPORT [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Maintenance and F5 Technical Support of the F5 code is provided only if the software (i) is unmodified;
```
Comment (mjmenger) [inline SUPPPORT]: the file should be renamed SUPPORT rather than SUPPPORT

File: ansible/roles/big-ip/defaults/main.yml [hunk 3]
```diff
@@ -1,23 +1,26 @@
 # role default variables
 elasticsearch_index: "f5telemetryindex"
 ts_listener_port: 6514
-log_pool: 10.1.20.9
+log_pool: "10.1.20.9"
```
Comment (mjmenger) [inline ansible/roles/big-ip/defaults/main.yml]: Should this be updated to a dynamic value? 10.1.0.0 isn't part of the VPC in the setup project.

Comment (aknot242) [reply ansible/roles/big-ip/defaults/main.yml]: This is a default (low-priority) value and my intent was that it was always meant to be overridden in a higher ranking variable definition. In our cases, we have always been overriding this value with the value that is declared in the inventory file.

Comment (aknot242) [reply ansible/roles/big-ip/defaults/main.yml]: Hopefully you're ok with this. Since the light appears to be green, i'll go ahead and approve. ;)

**Pull request merged**